Bulb/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
i think this is not related to this PR
may be ur PR is not up to date

Reply:
Bhai bulb wali meri pichli pr mein thi jo pr hochuki hai ab ye jo digital bnai hai esme meine bas image change ki hai code ko kuch nahi badla. Image bekar thi usme. (Naam digital wrist watch rakhna tha par wo hua nahi kuch error dikhara tha esliye digital rakhdia wo shayad mentor ne digital watch se change bhi kardia)

Reply:
@Vamp-harshit update your branch of PR

Reply:
ur current branch is 
 3 commits ahead, 21 commits behind arpit456jain:master.

Reply:
upadte ur branch by 
git pull origin master